lms/resources/_js_config/__init__.py

```diff
@@ -427,78 +427,34 @@ def _groups(self):
         # to the sync API to dynamically get the relevant groupings.
         return "$rpc:requestGroups"
 
-    def _canvas_sync_api(self):
-        req = self._request
-        sync_api_config = {
-            "authUrl": req.route_url("canvas_api.oauth.authorize"),
-            "path": req.route_path("canvas_api.sync"),
-            "data": {
-                "lms": {
-                    "tool_consumer_instance_guid": self._context.lti_params[
-                        "tool_consumer_instance_guid"
-                    ],
-                    "product": self._request.product.family,
-                },
-                "course": {
-                    "context_id": self._context.lti_params["context_id"],
-                    "custom_canvas_course_id": self._context.lti_params[
-                        "custom_canvas_course_id"
-                    ],
-                },
-                "assignment": {
-                    "resource_link_id": self._context.lti_params["resource_link_id"],
```

Comment:
No need for resource_link_id either, the views used it to get the group_set_id but that happens here now.